website/docs/r/container_group.html.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_group"
+sidebar_current: "docs-azurerm-resource-container-group"
+description: |-
+  Create as an Azure Container Group instance.
+---
+
+# azurerm\_container\_group
+
+Create as an Azure Container Group instance.
+
+## Example Usage (Linux)
+
+```hcl
+resource "azurerm_resource_group" "aci-rg" {
+  name     = "aci-test"
+  location = "west us"
+}
+
+resource "azurerm_container_group" "nginx" {
+  
+  name = "mynginx"
+  location = "west us"
+  resource_group_name = "${azurerm_resource_group.aci-rg.name}"
+  ip_address_type="public"
+  os_type = "linux"
+
+  container {
+    name = "nginx1"
+    image = "nginx:latest"
+    cpu ="1"
+    memory = "1.5"
+    port = "80"
+  }
+
+  tags {
+    environment = "testing"
+  }
+}
+```
+
+## Example Usage (Windows)

```

Comment:
given the only difference between these two is the OS field - I think we can just use the Linux example (and just rename that to `Example Usage`)?

Reply:
I've updated the linux example to use two containers.

Reply:
could we consolidate this into a single example, as they're fundamentally the same (compared to say Container Service in which some fields aren't valid for a non-Kubernetes cluster)?

Reply:
you mean two container groups in one file?

Reply:
I was referring to picking either the Windows or the Linux example and removing the other (since they're virtually the same)

Reply:
Done.